includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -131,6 +132,8 @@ function get_sortable_columns() {
 			case 'courses':
 				$columns = array(
 					'title'           => array( 'title', false ),
+					'students'        => array( 'students', false ),
+					'lessons'         => array( 'lessons', false ),
 					'completions'     => array( 'completions', false ),
 					'average_percent' => array( 'average_percent', false ),
```

Comment:
Looks like `average_percent` should be removed too since it looks like you renamed it to `average_grade`, and also because it didn't sort properly before that either. 😄

Reply:
I have a feeling I messed this up when merging stuff, not sure how it appeared here. I will remove everything except the title. Completions and percent also don't work I've checked it.